fireflower/targets.py

```diff
@@ -28,19 +28,33 @@ class FireflowerS3Target(S3Target):
 
     fs = None
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.local_s3_path = os.getenv('LOCAL_S3_PATH', None)
```

Comment:
actually, making this a class variable makes it hard to test, as the class will then read in the variable, before mocking.